Content.Server/GameObjects/Components/Items/Storage/ServerStorageComponent.cs

```diff
@@ -343,6 +365,7 @@ public override void ExposeData(ObjectSerializer serializer)
 
             serializer.DataField(ref _storageCapacityMax, "capacity", 10000);
             serializer.DataField(ref _occludesLight, "occludesLight", true);
+            serializer.DataField(ref _quickInsert, "quickInsert", false);
```

Comment:
i'd add a var for disabling the areapickup too

Reply:
Was just copying how it had been done in SS13, but I don't see why not.